keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -155,17 +195,19 @@ Provider can safely ignore `stdin` since input object doesn't carry any data.
 }
 ```
 
-`EXPIRATION` contains the RFC3339 timestamp with credential expiry. Client can
-cache provided credentials until this time.
+`expirationTimestamp` specifies the RFC3339 timestamp with credential expiry.
+Client can cache provided credentials in-memory until this time.  If no
+`expirationTimestamp` is provided, credentials will be cached in-memory
+throughout the runtime of the client (no attempt is made to infer an expiration
+time based on the credentials themselves).
 
-After `EXPIRATION`, client must execute the provider again for any new
-connections. For `client_key` mode, this applies even if returned certificate
-is still valid.
+After `expirationTimestamp`, client must execute the provider again for any new
+connections. For mTLS connections, this applies even if returned certificate
+is still valid (i.e. the `NotAfter` date is ignored).
```

Comment:
nit: clarify that existing connections can stay open

Reply:
Done.